### ISSUE TITLE
Collapse component is not accessible friend

### ISSUE THREAD

**Issue by kossel** [react-component/collapse#83]
Even there are some aria attributes, it still missing `tabIndex="0"` attributes and some keyPress event handling. So you can not navigate, collapse or expand the tabs using keyboard only.

Please have a look at
[https://springload.github.io/react-accessible-accordion/](https://springload.github.io/react-accessible-accordion/)

vs 

[http://react-component.github.io/collapse/examples/simple.html](http://react-component.github.io/collapse/examples/simple.html)

I think can PR if you accept this feature

**Comment by yesmeck** [comment]
Ref https://github.com/ant-design/ant-design/issues/6017

@kossel Welcome PR

**Comment by kossel** [comment]
Just want to make sure, looks like it can be done from rc-collapse or antd, where should I PR?

**Comment by yesmeck** [comment]
rc-collapse

**Comment by yesmeck** [comment]
antd is based on rc-collapse
